app/src/main/res/xml/debug_settings.xml

```diff
@@ -79,4 +79,8 @@
         android:title="@string/create_error_notification"
         app:singleLineTitle="false"
         app:iconSpaceReserved="false" />
+    <Preference
+        android:key="@string/reset_settings"
+        android:title="@string/settings_category_reset_title"
+        app:iconSpaceReserved="false" />
```

Comment:
We need to ensure that the title is fully displayed on small devices:
```suggestion
        app:singleLineTitle="false"
        app:iconSpaceReserved="false" />
```

Reply:
I see. Thank you for pointing it out. Should I commit the changes now?

Reply:
> I see. Thank you for pointing it out. I have committed the changes.

I think you can start work on the solution TobiGr mentioned above. I will help you.

Reply:
Since the deadline is passed and we have more assignments to do, also 4 final exams and a 2000-word essay are coming. We have to wait until 15 Nov or later to continue to work on this PR

Reply:
Ok, don't worry ;-)

I will turn this PR into a draft for now, feel free to write a comment once you get back on it, so that we see it

Reply:
Understood. Thank you.